drivers/soundwire/intel.c

```diff
@@ -1508,6 +1508,9 @@ static int intel_resume(struct device *dev)
 		return ret;
 	}
 
+	/* make sure all Slaves are tagged as UNATTACHED */
+	sdw_clear_slave_status(&sdw->cdns.bus);
```

Comment:
ah darn it, yes this was missing. Again a copy/paste error, rather not doing a copy/paste....

Thanks @bardliao I will apply this asap to the integration/soundwire-intel branch.

Nice catch!